test/db-journey-polyline.js

```diff
@@ -23,7 +23,7 @@ const opt = {
 	bike: false,
 	tickets: true,
 	polylines: true,
-	remarks: true,
+	remarks: false,
```

Comment:
I assume that you change `opt.remarks` to `false` whenever adding the expected remarks to the fixture(s) would have been too cumbersome?

Reply:
Yes, at least it was easier to set it to `false`, and I figured it would not be a bad idea to have some variance and also test the `false` behaviour somewhere